Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.11
+ADD . /usr/src/nimbess-agent
+RUN mkdir -p $GOPATH/src/github.com/nimbess/nimbess-agent
+WORKDIR $GOPATH/src/github.com/nimbess/nimbess-agent
+COPY . .
```

Comment:
What does the ADD . up earlier do if there is another COPY .  . here? Seems like this COPy here is the important piece since you build the binary from it.

Reply:
I copied this from another project. We don't need it I don't think. Will remove.